site2/docs/reference-configuration.md

```diff
@@ -802,6 +809,7 @@ The [Pulsar proxy](concepts-architecture-overview.md#pulsar-proxy) can be config
 |tlsRequireTrustedClientCertOnConnect|  Whether client certificates are required for TLS. Connections are rejected if the client certificate isn’t trusted. |false|
 |tlsProtocols|Specify the tls protocols the broker will use to negotiate during TLS Handshake. Multiple values can be specified, separated by commas. Example:- ```TLSv1.3```, ```TLSv1.2``` ||
 |tlsCiphers|Specify the tls cipher the broker will use to negotiate during TLS Handshake. Multiple values can be specified, separated by commas. Example:- ```TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256```||
+|tlsProvider| The TLS provider for the broker service. <br />When TLS authentication with CACert is used, the valid value is either `OPENSSL` or `JDK`.<br />When TLS authentication with KeyStore is used, available options can be `SunJSSE`, `Conscrypt` and so on. | N/A
```

Comment:
N/A -> Conscrypt  ?

Reply:
Seems it is left blank -->  https://github.com/apache/pulsar/pull/13740/files#diff-f6021f4eb78412cb22fddd64b711e74ef794d071f45200a1997a9e682077895aL79. 
@nodece Can you pls double-check the default tls provider for proxy? For websocket, the default one is Conscrypt.

Reply:
@Technoboy- There's no problem here. This tlsProvider is only used to broker service, the web service requires the Conscrypt, we can config this by the `webserviceTlsProvider`.